### PR TITLE
Uswds Homepage Update

### PR DESCRIPTION
**Pull request by ojbravo**
Added Hero Section and updated TOC styling on homepage

### CODE REVIEW

File: src/homepage/ForDataUsers/ResearchAndReports.jsx [hunk 1]
```diff
@@ -4,13 +4,14 @@ export const ResearchAndReports = ({ isProdBeta }) => {
...
-          Research and Reports
-        </ExternalLink>
-      </h3>
+    <article>
+      <h3>Research and Reports</h3>
       <p>Research and reports on mortgage market activity.</p>
-    </header>
+      <ul>
+        <li>
+          <a href="https://www.consumerfinance.gov/data-research/hmda/" target="_blank">Research and Reports</a>
```
Comment (Michaeldremy) [inline src/homepage/ForDataUsers/ResearchAndReports.jsx:12]: Before there was an external icon to let users know they were going to leave the site, can we re-add that?

Replace the `a` tag with...

```js
<ExternalLink url='https://www.consumerfinance.gov/data-research/hmda/'>
  Research and Reports
</ExternalLink>
```

File: src/homepage/index.js [hunk 2]
```diff
@@ -19,6 +23,9 @@ const Home = ({ config }) => {
   const isProdBeta = isProd() && isBeta()
```
Comment (Michaeldremy) [inline src/homepage/index.js:23]: `isProdBeta` is not being used with the new changes. The benefit of this flag was to display certain items in prod beta on the homepage. Can we re-add the flag to the new components and hide the prod beta links?

Previous link or content hidden on prod beta:

Filing doc link
```JS
{hideContent && (
  <li>
    <a href='/documentation/category/filing-instructions-guide'>
      HMDA Online FIG
      <NewIndicator />
    </a>
  </li>
)}
```

`Info for Data Users` column is completely hidden on prod beta.

File: src/homepage/index.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
 import React from "react"
 import "./ColumnLayout.css"
 import "./Home.css"
+import { Hero } from "./Hero"
+import { QuickLinks } from "./QuickLinks"
 import { AnnouncementBanner } from "./AnnouncementBanner"
+import { FilerInfo } from "./ForFilers/FilerInfo"
+import { DataInfo } from "./ForDataUsers/DataInfo"
 import { ColumnLayout, Column } from "./ColumnLayout.jsx"
```
Comment (Michaeldremy) [inline src/homepage/index.js:9]: Lines 9 - 20 has a lot of unused components/code. If we are certain we won't be using a lot of these components can we also remove these components from the codebase?

File: src/homepage/Hero.jsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { VideoModal } from './VideoModal.jsx';
+import iconSprite from "../common/uswds/img/sprite.svg";
+
+// const [showModal, setShowModal] = useState(false);
+
+// const handleShowModal = () => {
+//   setShowModal(true);
+// };
```
Comment (Michaeldremy) [inline src/homepage/Hero.jsx:9]: If the code is not needed, can we remove this code?

**Review by Michaeldremy — CHANGES_REQUESTED** [review]
Left some required changes. Happy to walk through and discuss these!

File: src/common/OptionCarousel.jsx [hunk 5]
```diff
@@ -45,7 +45,7 @@ export const OptionCarousel = ({
   const navControls = useAutoAdvance({ cycleTime, totalCount, autoAdvance })
 
   return (
-    <div id={id} className="oc" style={styles}>
+    <div id={id} className="oc" /*style={styles}*/>
```
[the remark this answers is not in the document]
Comment (ojbravo) [reply src/common/OptionCarousel.jsx:48]: I added the styles back in so the Carousel doesn't cause a content shift. You and Meissa win.

File: src/common/OptionCarousel.jsx
[the remark this answers is not in the document]
Comment (ojbravo) [reply src/common/OptionCarousel.jsx:1]: Fixed.

File: src/homepage/ForDataUsers/ResearchAndReports.jsx (hunk 1 above)
Comment (ojbravo) [reply src/homepage/ForDataUsers/ResearchAndReports.jsx:12]: Fixed.

File: src/homepage/Hero.jsx (hunk 4 above)
Comment (ojbravo) [reply src/homepage/Hero.jsx:9]: Code removed.

File: src/homepage/index.js (hunk 3 above)
Comment (ojbravo) [reply src/homepage/index.js:9]: I removed the unused imports as they are in the individual components (<DataInfo /> and <FilerInfo />)

File: src/homepage/index.js (hunk 2 above)
Comment (ojbravo) [reply src/homepage/index.js:14]: Fixed.

**Pull request merged**